apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/KeysTests.cs

```diff
@@ -39,23 +47,24 @@ public void CreateKeyFromParameterCollection()
                     { "", SpannerDbType.Timestamp, new DateTime(2021, 9, 10, 9, 37, 10, DateTimeKind.Utc) }
                 });
             var index = 0;
-            Assert.True(key.KeyParts.Values[index++].BoolValue);
-            Assert.Equal(Convert.ToBase64String(new byte[] {1,2,3}), key.KeyParts.Values[index++].StringValue);
-            Assert.Equal("2021-09-10", key.KeyParts.Values[index++].StringValue);
-            Assert.Equal(3.14, key.KeyParts.Values[index++].NumberValue);
-            Assert.Equal("1", key.KeyParts.Values[index++].StringValue);
-            Assert.Equal("{\"key\": \"value\"}", key.KeyParts.Values[index++].StringValue);
-            Assert.Equal("10.1", key.KeyParts.Values[index++].StringValue);
-            Assert.Equal("20.1", key.KeyParts.Values[index++].StringValue);
-            Assert.Equal("test", key.KeyParts.Values[index++].StringValue);
-            Assert.Equal("2021-09-10T09:37:10Z", key.KeyParts.Values[index++].StringValue);
+            var protobuf = key.ToProtobuf(s_conversionOptions);
```

Comment:
I would suggest just creating the expected value instead, e.g.

```csharp
var expected = new ListValue
{
    Values =
    {
        Value.ForBool(true),
        Value.ForString(Convert.ToBase64String(new byte[] { 1, 2, 3 })),
        Value.ForString("2021-09-10"),
        ...
    }
}
Assert.Equal(expected, actual);
```

Reply:
(That can come later though.)

Reply:
Yes, this that Jon is saying is what I meant in my earlier comment "obtain the protobuf value and make sure it is what you are expecting". `actual` would be the protobuf value. Sorry if I wasn't clear enough.